### ISSUE TITLE
FunctionClauseError calling PropCheck.StateM.Reporter.pretty_cmd_name/2 with a list

### ISSUE THREAD

**Issue by aherranz** [alfert/propcheck#171]
Function `counter_example_inspect` in `Properties`, calls `Reporter.pretty_print_counter_example_cmd(cmds)` with a list of commands. But `pretty_print_counter_example_cmd` expects just a command. Substituting call

```elixir
        Reporter.pretty_print_counter_example_cmd(cmds)
```

by 

```elixir
        for cmd <- cmds do
          Reporter.pretty_print_counter_example_cmd(cmd)
        end
```

should be enough.

**Comment by aherranz** [comment]
I PRed!

**Comment by alfert** [comment]
Resolved in #172
